go.work

```diff
@@ -0,0 +1,15 @@
+go 1.18
+
+use (
+	./api
+	./container
+	./cosmovisor
+	./db
+	./errors
+	./orm
+	./store/tools/ics23
+)
+
+replace github.com/gogo/protobuf v1.3.3 => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+replace github.com/cosmos/cosmos-sdk v0.46.0-alpha3 => ../
```

Comment:
when we cut releases, do we need to remove these replace tags to avoid releasing tools like cosmovisor with master of the sdk?

Reply:
and-- this part, I also don't know.

This is a very good question.

Reply:
I guess ideal would look like restoring the import to its former state?

I think that it may be possible.  Will try now